metadata.yaml

```diff
@@ -0,0 +1,13 @@
+name: "metadata"
+buildsystem: simple
+build-commands:
+  - "install -D com.deflemask.DefleMask.png /app/share/icons/hicolor/512x512/apps/com.deflemask.DefleMask.png"
+  - "install -D com.deflemask.DefleMask.desktop /app/share/applications/com.deflemask.DefleMask.desktop"
+  - "install -D com.deflemask.DefleMask.appdata.xml /app/share/metainfo/com.deflemask.DefleMask.appdata.xml"
+sources:
+  - type: file
+    path: "com.deflemask.DefleMask.png"
+  - type: file
+    path: "com.deflemask.DefleMask.desktop"
+  - type: file
+    path: "com.deflemask.DefleMask.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html